unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -1763,5 +1762,16 @@ public static void GetFriendRequests(int sentLimit, int sentSkip, int receivedLi
                 sentLimit = sentLimit
             });
         }
+        
+        public static void RequestAudioDevices() => SendMessage("RequestAudioDevices");
+
+        public static void SetInputAudioDevice(string inputDeviceId)
+        {
+            SendMessage(nameof(SetInputAudioDevice), new SetInputAudioDevicePayload()
```

Comment:
So in the end we can confirm that the `nameof` works as expected? Should we have another PR where we update everything to follow the same standard?

Reply:
of course `nameof` works as expected :) .NET standards suggest to use it everywhere where it is possible (because in this way you avoid re-naming issues ;) )
We can add it to our code style guideline